bmi_index.js

```diff
@@ -0,0 +1,28 @@
+const formElement = document.querySelector(".bmi-form");
+
+const heightInput = document.querySelector("#height");
+const weightInput = document.querySelector("#weight");
+
+const bmiMessage = document.querySelector(".result");
+
+// const button = document.querySelector("button");
+
+// function calculate(heightInput, weightInput) {
+//   let result = heightInput.value / weightInput.value ** 2;
+//   console.log(result);
+// }
+
+formElement.addEventListener("submit", (event) => {
+  const bmi = (weightInput.value / heightInput.value ** 2) * 10000;
```

Comment:
maybe there is a method to not get a very long bmi number?
For example right now I'm getting `27.75748721694668` as an result `27.76` would be way better ;-)

Reply:
You're right! :) There is a method => bmi.toFixed(2); 
Implemented it